src/main/java/org/opensearch/ad/model/AnomalyResult.java

```diff
@@ -169,28 +174,47 @@
     71604.0,
     75236.0
     ],
-    Since relativeIndex is -2, we look back baseDimension * 2 and get the oldValues:
+    Since rcf returns relativeIndex is -2, we look back baseDimension * 2 and get the pastValues:
     "startOfAnomaly": true,
     "inHighScoreRegion": true,
-    "relativeIndex": -2,
-    "oldValues": [
+    "relevantValues": [
     17265.0,
     4113.142857142857,
     49.0,
     86376.0,
     64878.0
     ],
     
-    So oldValues is null when relativeIndex is 0 or startOfAnomaly is true.*/
-    private final double[] oldValues;
+    So pastValues is null when relativeIndex is 0 or startOfAnomaly is true
+    or the current shingle is not an anomaly.
+    
+    In the UX, if pastValues value is null, we can just show attribution/expected
+    value and it is implicit this is due to current input; if pastValues is not
+    null, it means the the attribution/expected values are from an old value
+    (e.g., 2 steps ago with data [1,2,3]) and we can add a text to explain that.
+    */
+    private final List<DataByFeatureId> pastValues;
```

Comment:
had a new thought here, would it actually make sense to just include the relevant values here? (rename to `relevantValues`), and include the current values if `relativeIndex` is 0, else populate with past values?

Few reasons for this:
1. unifies the data in the results (always have values in here, regardless of if the anomaly is found late or not)
2. simplifies logic needed from external users building their own dashboards (+ frontend plugin) on getting the current values. it can be retrieved directly from here, rather than having to use the detector's interval and shingle size to calculate the list to retrieve

Reply:
^ this would be redundant, since the current values would already be what's included in the feature values, and no extra data fetching would be needed. the current `pastValues` makes more sense here. please disregard.

Reply:
As discussed offline, it would be redundant if we have both feature_data and relevantValues since relevantValues is feature_data when there is no anomaly or relative index is 0. if removing feature_data, there would be bwc issue. Also, the front end needs a lot of modification.  if keeping feature_data, our index size will increase.